Raspi/Test/mock_wireless_module.py

```diff
@@ -0,0 +1,135 @@
+import time
+import random
+import argparse
+import json
+import paho.mqtt.client as mqtt
+
+
+parser = argparse.ArgumentParser(
+    description='MQTT wireless sensor test script that sends fake data',
+    add_help=True)
+
+parser.add_argument(
+    '-t', '--time', action='store', type=int,
+    default=float('Inf'), help="""Length of time to send data in seconds
+    (duration). If nothing is selected it will continuously send out data.""")
+
+parser.add_argument(
+    '-r', '--rate', action='store', type=float, default=1,
+    help="""Rate of data in data sent per second. Default is 1 data pack sent
+    per second.""")
+
+parser.add_argument(
+    '--host', action='store', type=str, default="192.168.100.100",
+    help="""Address of the MQTT broker. If nothing is selected it will
+    default to 192.168.100.100.""")
+
+parser.add_argument(
+    '--id', action='store', type=int, default=None,
+    help="""Specify the sensor to produce fake data. eg. --id 1 specifies that
+    sensor 1 only produces data. If nothing is given all sensors will be
+    active.""")
+
+
+def send_fake_data(client, duration, rate, sensorID):
+    """ Send artificial data over MQTT for each sensor chanel. Sends [rate] per
+    second for [duration] seconds"""
+
+    start_time = round(time.time(), 2)
+
+    while True:
+        current_time = round(time.time(), 2)
+        total_time = round(current_time - start_time, 2)
+
+        print()     # Newline for clarity
+
+        # Wireless Sensor 1 (Middle)
+        if sensorID == 1 or sensorID is None:
+
+            # generate JSON data and publish it over MQTT
+            data = {
+                "CO2": Sensors.CO2(),
+                "temperature": Sensors.temperature(),
+                "humidity": Sensors.humidity(),
+                "accelerometer": Sensors.accelerometer(),
+                "gyroscope": Sensors.gyroscope(),
+                "battery": Sensors.battery(),
+                "time": Sensors.time()}
+
+            JSON_data = json.dumps(data)
```

Comment:
`snake_case` (sorry this is probably annoying lol)

Reply:
It's good it makes it more readable, thanks for pointing it out